cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -343,6 +340,14 @@ def get_image_format_from_buf(qhdr):
     return "raw"
 
 
+def get_hasher(algorithm, default="md5"):
+    if algorithm == "xxh3":
+        return xxhash.xxh3_64()
+
+    if algorithm not in hashlib.algorithms_available:
+        algorithm = default
+    return eval("hashlib.%s()" % algorithm)
```

Ruff (0.14.1)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)